tests/ClientTest.php

```diff
@@ -534,43 +357,268 @@ public function testAddBreadcrumbStoresBreadcrumbReturnedByBeforeBreadcrumbCallb
+                                'value' => 'foo',
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+            [
+                new \RuntimeException('foo'),
+                [
+                    'auto_log_stacks' => false,
+                ],
+                [
+                    'level' => Severity::ERROR,
+                    'exception' => [
+                        'values' => [
+                            [
+                                'type' => \RuntimeException::class,
+                                'value' => 'foo',
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+            [
+                new \ErrorException('foo', 0, E_USER_WARNING),
+                [],
+                [
+                    'level' => Severity::WARNING,
+                    'exception' => [
+                        'values' => [
+                            [
+                                'type' => \ErrorException::class,
+                                'value' => 'foo',
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+            [
+                new \BadMethodCallException('baz', 0, new \BadFunctionCallException('bar', 0, new \LogicException('foo', 0))),
+                [
+                    'excluded_exceptions' => [\BadMethodCallException::class],
+                ],
+                [
+                    'level' => Severity::ERROR,
+                    'exception' => [
+                        'values' => [
+                            [
+                                'type' => \LogicException::class,
+                                'value' => 'foo',
+                            ],
+                            [
+                                'type' => \BadFunctionCallException::class,
+                                'value' => 'bar',
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+        ];
     }
 
-    private function createCarelessExceptionWithStacktrace()
+    public function testConvertExceptionContainingLatin1Characters()
     {
-        try {
-            throw new CarelessException('Foo bar');
-        } catch (\Exception $ex) {
-            return $ex;
-        }
+        $options = new Options(['mb_detect_order' => ['ISO-8859-1', 'ASCII', 'UTF-8']]);
+
+        $utf8String = 'äöü';
+        $latin1String = utf8_decode($utf8String);
+
+        /** @var TransportInterface|\PHPUnit_Framework_MockObject_MockObject $transport */
+        $transport = $this->createMock(TransportInterface::class);
+
+        $transport->expects($this->once())
+            ->method('send')
+            ->with($this->callback(function (Event $event) use ($utf8String) {
+                $this->assertNotNull($event);
```

Comment:
This assert is unnecessary because the argument is typehinted to not accept `null` values